framer/DOMEventManager.coffee

```diff
@@ -42,6 +42,11 @@ class exports.DOMEventManager
 
 		@_elements[element._eventManagerId]
 
+	remove: (element) =>
+		return unless element._eventManagerId
+		delete @_elements[element._eventManagerId]
+		element._eventManagerId = 0
```

Comment:
shouldn't we set it to `null` instead of 0 here, to more explicitly specify unsetting it?

Reply:
`0` is a perfectly good `null` for number types, the advantage is this way the field doesn't switch type, which might help the javascript engine keep the code faster. Also why I made sure counting starts at `1`, before this the first layer using `wrap()` would get two `DOMEventManagerElement`s ...